Services/ReportService.cs

```diff
@@ -1017,7 +1017,139 @@ public IWorkbook GenerateAttendanceReportWorkbook(AttendanceReportModel attendan
+        private void GenerateOASWorksheetBodyLikeTerrain(IWorksheet sheet, IList<IGrouping<string, OASWorksheetAnswers>> groupedAnswers, ref int rowNumber, ref int columnNumber)
+        {
+            var pdrText = "";
+            var titleRow = rowNumber;
+            int pdrStartRow = 99;
+            foreach (var plan in groupedAnswers.FirstOrDefault().OrderBy(ga => ga.InputTitleSortIndex).ThenBy(ga => ga.InputSortIndex).ToList())
+            {
+                // Plan Do Review
+                rowNumber++;
+                if (pdrText != plan.InputTitle)
+                {
+                    pdrText = plan.InputTitle;
+                    sheet.Range[rowNumber, columnNumber].Text = plan.InputTitle;
+                    sheet.Range[rowNumber, columnNumber].CellStyle.Color = GetInputTitleColour(plan.InputTitle);
+                    sheet.Range[rowNumber, columnNumber].BorderAround();
+                    sheet.Range[rowNumber, columnNumber].CellStyle.Rotation = 90;
+                    sheet.Range[rowNumber, columnNumber].CellStyle.Font.Bold = true;
+                    sheet.Range[rowNumber, columnNumber].CellStyle.HorizontalAlignment = ExcelHAlign.HAlignCenter;
+                    if (pdrStartRow > rowNumber)
+                        pdrStartRow = rowNumber;
+                    else
+                    {
+                        sheet.Range[pdrStartRow, columnNumber, rowNumber - 1, columnNumber].Merge();
+                        sheet.Range[pdrStartRow, columnNumber, rowNumber - 1, columnNumber].BorderAround();
+                        pdrStartRow = rowNumber;
+                    }
+                }
+
+                // iStatement
+                sheet.Range[rowNumber, columnNumber + 1].Text = plan.InputLabel;
+                sheet.Range[rowNumber, columnNumber + 1].BorderAround();
+                sheet.Range[rowNumber, columnNumber + 1].CellStyle.Font.Bold = true;
+                sheet.Range[rowNumber, columnNumber + 1].CellStyle.HorizontalAlignment = ExcelHAlign.HAlignCenter;
+                sheet.Range[rowNumber, columnNumber + 1].CellStyle.Rotation = 0;
+                sheet.Range[rowNumber, columnNumber + 1].CellStyle.WrapText = true;
+                sheet.SetColumnWidth(columnNumber + 1, 50);
+                sheet.AutofitRow(rowNumber);
+            }
+
+            // Member Name and answers
+            columnNumber++;
+            sheet.Range[titleRow + 1, columnNumber + 1].FreezePanes();
+            rowNumber = titleRow;
+            foreach (var groupedAnswer in groupedAnswers.OrderBy(ga => ga.Key))
+            {
+                columnNumber++;
+                sheet.Range[rowNumber, columnNumber].Text = groupedAnswer.Key;
+                sheet.Range[rowNumber, columnNumber].BorderAround();
+                sheet.Range[rowNumber, columnNumber].CellStyle.Font.Bold = true;
+
```

Comment:
Can we rotate the names 90 so the page doesn't get too wide with about 20 kids.
```
                sheet.Range[rowNumber, columnNumber].CellStyle.Rotation = 90;
                sheet.Range[rowNumber, columnNumber].CellStyle.HorizontalAlignment = ExcelHAlign.HAlignCenter;

```

Reply:
Done